src/main/java/org/radarbase/output/storage/LocalStorageDriver.kt

```diff
@@ -24,21 +25,12 @@ import java.nio.file.Files
 import java.nio.file.Path
 import java.nio.file.StandardCopyOption.ATOMIC_MOVE
 import java.nio.file.StandardCopyOption.REPLACE_EXISTING
-import java.nio.file.attribute.FileAttribute
-import java.nio.file.attribute.PosixFileAttributes
 import java.nio.file.attribute.PosixFilePermissions
 
-class LocalStorageDriver : StorageDriver {
-    private var uid = -1
-    private var gid = -1
-
-    override fun init(properties: Map<String, String>) {
-        uid = properties["localUid"]?.toIntOrNull() ?: -1
-        gid = properties["localGid"]?.toIntOrNull() ?: -1
-
+class LocalStorageDriver(private val config: LocalConfig) : StorageDriver {
```

Comment:
Likewise, these can be called Target instead of driver. Like `LocalTargetStorage` or `LocalDestinationStorage` ?